### PR TITLE
FEAT: Added sign-up page

### PR DESCRIPTION
**Pull request by kotesh-arya**
Screenshot from 2022-04-04 22-58-36

### CODE REVIEW

File: src/pages/SignUp/SignUp.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.signup-container {
+    text-align: left;
+    width: 25%;
+    text-align: center;
```
Comment (MaruthiPothuganti) [inline src/pages/SignUp/SignUp.css:3]: text-align is already present above.

**Review by MaruthiPothuganti — COMMENTED** [review]
A change is recommended. Apart from that, Code is clean and  good to go.

File: src/pages/SignUp/SignUp.css (hunk 1 above)
Comment (kotesh-arya) [reply src/pages/SignUp/SignUp.css:3]: OH!, will update it 😁

File: src/pages/SignUp/SignUp.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    box-shadow: 2px 2px 10px var(--primary-color);
+    margin: auto;
+    margin-top: 10rem;
+    margin-bottom: 5.5rem;
```
Comment (zabihshaik) [inline src/pages/SignUp/SignUp.css]: Better to use short-hand property for margin

File: src/pages/SignUp/SignUp.jsx [hunk 3]
```diff
@@ -1,10 +1,36 @@
...
-
+import {  Footer, NavBar } from '../HomePage/HomePageComponents'
+import { Link } from 'react-router-dom'
+import '../SignUp/SignUp.css'
 function SignUp() {
 return (
 <div>
-    <h1>Blank SignUp page for Route checking</h1>
+    <NavBar/>
+    {/* <h1>Blank SignUp page for Route checking</h1> */}
```
Comment (zabihshaik) [inline src/pages/SignUp/SignUp.jsx]: Better to remove commented code.

**Review by zabihshaik — APPROVED** [review]
Everything. Just a couple of comments. Plz check.

**Review by Rockyraven — APPROVED** [review]
do some minar change 
otherwise your code is clean and easy to readable

File: src/pages/SignUp/SignUp.css (hunk 2 above)
Comment (kotesh-arya) [reply src/pages/SignUp/SignUp.css]: Yeah, Sure 👍🏻

File: src/pages/SignUp/SignUp.jsx (hunk 3 above)
Comment (kotesh-arya) [reply src/pages/SignUp/SignUp.jsx]: 👍🏻

File: src/pages/SignUp/SignUp.css [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+.check-box {
+    height: 1.5rem;
+    width: 1.5rem;
+    margin-right: 10px;
+}
+
+.text-checkbox {
+    display: flex;
+    flex-flow: row wrap;
```
Comment (vinc3nati) [inline src/pages/SignUp/SignUp.css:35]: nice use of short hand property

**Review by vinc3nati — APPROVED** [review]

**Pull request merged**